kingpin/actors/aws/ecs.py

```diff
@@ -406,7 +420,369 @@ def _execute(self):
         if self.option('wait'):
             yield self._wait_for_tasks(tasks)
         else:
-            self.log.info('Not waiting for tasks to stop')
+            self.log.info('Not waiting for tasks to complete')
+
+
+# http://boto3.readthedocs.io/en/latest/reference/services/ecs.html
+SERVICE_DEFINITION_SCHEMA = {
+    'type': 'object',
+    'additionalProperties': False,
+    'properties': {
+        'loadBalancers': {
+            'type': 'array',
+            'items': {
+                'type': 'object',
+                'additionalProperties': False,
+                'properties': {
+                    'loadBalancerName': {'type': 'string'},
+                    'containerName': {'type': 'string'},
+                    'containerPort': {'type': 'number'},
+                }
+            }
+        },
+        'role': {'type': 'string'},
+        'deploymentConfiguration': {
+            'type': 'object',
+            'additionalProperties': False,
+            'properties': {
+                'maximumPercent': {'type': 'number'},
+                'minimumHealthyPercent': {'type': 'number'},
+            }
+        }
+    }
+}
+
+
+class DeployService(ECSBaseActor):
```

Comment:
I would rename this to `Service`. You're really ensuring that a given service is running with a given task definition. If you were simply "deploying" the service (where a task definition had already been defined), then I can see this being named `DeployService`.